app/src/main/java/io/zenandroid/onlinego/ui/screens/puzzle/TsumegoAction.kt

```diff
@@ -0,0 +1,31 @@
+package io.zenandroid.onlinego.ui.screens.puzzle
+
+import android.graphics.Point
+import io.zenandroid.onlinego.data.model.local.Puzzle
+import io.zenandroid.onlinego.data.model.local.PuzzleCollection
+
+sealed class TsumegoAction {
```

Comment:
minor: in the latest iteration of the Kotlin language this should be a "sealed Interface" and the actual objects and classes should be "data object" or "data classes". But it doesn't really matter that much.

Reply:
no worries, will change